staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1160,6 +1160,12 @@ type KubeVirtSpec struct {
 	// holds kubevirt configurations.
 	// same as the virt-configMap
 	Configuration KubeVirtConfiguration `json:"configuration,omitempty"`
+
+	// selectors and tolerations that should apply to KubeVirt infrastructure components
+	Infra *HyperConvergedConfig `json:"infra,omitempty"`
```

Comment:
@tiraboschi pointers for Infra and Workloads too...

Reply:
is there a reason pointers are used here?

Reply:
a couple of reasons.

if these are static there's junk/nonsense lines in the kubevirt-cr `tolerations: {}` -- so it's cleaner and less confusing to admins.

If this is a pointer, the operator can short circuit the entire merge process at a glance. if this is static, there's no obvious queue that no values are/are not set and each one must be checked.